rollup/transaction_ingestion_sql_test.go

```diff
@@ -0,0 +1,231 @@
+package rollup
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+var db *sqlx.DB
+var err error
+
+func init() {
+	conn := "user=test password=test sslmode=disable dbname=rollup"
+
```

Comment:
This should be refactored to use https://github.com/onsi/ginkgo for easier management of the tests here. It gives mocha like before and after hooks.